Source/core/settings/ChromeSettingsProvider.js

```diff
@@ -67,11 +77,15 @@ ChromeSettingsProvider.prototype.get = function(key, def) {
  * returns carbon emission rate
  * default emission rate provided by:
  * https://www.epa.gov/sites/production/files/2016-02/documents/420f14040a.pdf
+ * find a credible source for public transport emission in kg/hr
  * @return {number}
  */
 
-ChromeSettingsProvider.prototype.getCarbonEmission = function() {
-  return this.get('emissionRate', 0.255384);
+ChromeSettingsProvider.prototype.getCarbonEmission = function(type) {
+  switch(type){
+  case 'd': return this.get('emissionRate', 0.255384);
+  case 't': return 1;
```

Comment:
Are you assuming that the carbon emission rate for public transportation is equal to 1kg/km ?